Makefile.js

```diff
@@ -663,11 +663,14 @@ target.gensite = function(prereleaseVersion) {
                 }
                 text += "* [Documentation source](" + docsUrl + baseName + ")\n";
             }
+
+            // 9. Update content of the file with changes
+            text.to(filename.replace("README.md", "index.md"));
```

Comment:
I removed that line because I also removed `README.md` from under `rules` directory. Or is this from some other directory?